ee/src/main/java/org/jboss/as/ee/concurrent/ControlPointUtils.java

```diff
@@ -172,22 +172,36 @@ public void run() {
             if (controlPoint == null) {
                 runnable.run();
             } else {
+                Exception runnableException = null;
                 try {
                     if (controlPoint.beginRequest() == RunResult.RUN) {
                         try {
                             runnable.run();
+                        } catch (Exception e) {
+                            runnableException = e;
+                            throw e;
                         } finally {
                             controlPoint.requestComplete();
                         }
                     } else {
                         throw EeLogger.ROOT_LOGGER.cannotRunScheduledTask(runnable);
                     }
                 } catch (RuntimeException re) {
-                    EeLogger.ROOT_LOGGER.failedToRunTask(re);  // note that this is now a DEBUG message as we rethrow re
-                    throw re;
+                    // WFLY-13043
+                    EeLogger.ROOT_LOGGER.failedToRunTask(runnable,re);
+                    if (runnableException == null) {
+                        return;
+                    } else {
+                        throw EeLogger.ROOT_LOGGER.failureWhileRunningTask(runnable, re); // note that this is now a DEBUG message as we rethrow e
+                    }
                 } catch (Exception e) { // TODO when WFCORE-6153 is done, drop this block. The actual try code only throws RuntimeExceptions
```

Comment:
WFCORE-6153 is now done and merged, so please just drop this entire catch block.